lib/plugins/aws/lib/naming.js

```diff
@@ -220,6 +220,10 @@ module.exports = {
     return `${this.getNormalizedFunctionName(functionName)}LambdaPermission${
       this.normalizeTopicName(topicName)}SNS`;
   },
+  getLambdaSnsSubscriptionLogicalId(functionName, topicName) {
```

Comment:
And an entry in the table of `logicalIds` would be great: https://github.com/serverless/serverless/blob/master/docs/providers/aws/guide/resources.md